src/Gizmos/positionGizmo.ts

```diff
@@ -87,6 +87,18 @@ export class PositionGizmo extends Gizmo {
             }
         });
     }
+
+    /**
+     * True when the mouse pointer is hovering a gizmo mesh
+     */
+    public get isHovered() {
+        var hovered = false;
+        [this.xGizmo, this.yGizmo, this.zGizmo, this.xPlaneGizmo, this.yPlaneGizmo, this.zPlaneGizmo].forEach((gizmo) => {
+            hovered = hovered || gizmo.isHovered;

```

Comment:
This creates an array which is immediately GC each time `isHovered` is called (same thing below for the other gizmo classes).

I don't know how bad it is given the very small size of the array(?)

Reply:
I will revisit that in the future.